pkg/containertools/runner.go

```diff
@@ -130,10 +130,14 @@ func (r *ContainerCommandRunner) Unpack(image, src, dst string) error {
 	r.logger.Infof("running %s create", r.containerTool)
 	r.logger.Debugf("%s", command.Args)
 
-	out, err := command.CombinedOutput()
+	out, err := command.Output()
 	if err != nil {
-		r.logger.Errorf(string(out))
-		return fmt.Errorf("error creating container %s: %v", string(out), err)
+		msg := err.Error()
+		if exitErr, ok := err.(*exec.ExitError); ok {
+			msg = fmt.Sprintf("%s: %s", err, exitErr.Stderr)
+		}
+		r.logger.Errorf(msg)
```

Comment:
nit: would logging the error here and then bubbling it up cause the message to occur twice in a row in the command output?

Reply:
Yeah, generally speaking, logging at the source and then returning an error can lead to duplicate error logging, so if we can double-check how we're handling logging at the various call sites, and instead bubble the error up that should be sufficient.